Docs: Small fixups related to running agents in AWS (#2522)
guardicore/monkey: docs/content/usage/integrations/aws-run-on-ec2-machine.md

## When to use this feature

This feature allows execution of the Infection Monkey agent on EC2 instances with the help of an SSM agent.

> Alternatively, you will need to execute a command on an EC2 instance manually (see [running manually](../../running-manually)) or using other management tools.

Reviewer: ```suggestion
Alternatively, you can execute a command on an EC2 instance manually (see [running manually](../../running-manually)) or using other management tools.
```
